package.json

```diff
@@ -17,7 +17,9 @@
     "next": "13.0.3",
     "react": "18.2.0",
     "react-dom": "18.2.0",
-    "styled-components": "^5.3.6"
+    "styled-components": "^5.3.6",
+    "swr": "^1.3.0",
+    "use-router": "^0.4.2"
```

Comment:
I'm wondering, what is this package `use-router` and what is it for? I don't think you need it. Did you install it by accident?